.github/workflows/packaging.yml

```diff
@@ -75,12 +141,18 @@ jobs:
       - name: Build and test
         shell: bash
         run: |
-          docker-compose run docs
+          pushd adbc
+          docker-compose run \
+            -e SETUPTOOLS_SCM_PRETEND_VERSION=${VERSION} \
```

Comment:
I'll follow up and make sure this actually makes it into the docs (the Sphinx config tries its best to _not_ assume that any Python modules are installed)

Reply:
Ah, I didn't consider the docs version.
It's just for building Python modules that are used in the docs. If we don't specify it, built Python modules have invalid version and can't be installed.

Reply:
Ah, right.

The main thing I want to make sure of is that the nightly build still has the "(dev)" version, or else it'll overwrite the docs for a release. I guess that means we should undo the changes in `conf.py` here, since website_build.sh uses the version from conf.py to decide whether it's dev or release docs.

Reply:
OK.
Can we use `0.1.0 (dev)`/`0.1.0-SNAPSHOT` in this pull request? Or should we use `1.0.0 (dev)`/`1.0.0-SNAPSHOT`?

Reply:
`0.1.0 (dev)` (or SNAPSHOT) is fine - the post release process will bump it to `0.2.0 (dev)`

Reply:
OK. Updated.